modules/RTC/TraceablePeerConnection.js

```diff
@@ -1689,6 +1778,29 @@ const _fixAnswerRFC4145Setup = function(offer, answer) {
 
 TraceablePeerConnection.prototype.createAnswer
 = function(successCallback, failureCallback, constraints) {
+    if (RTCBrowserType.supportsRtpSender()
+            && this.isSimulcastOn()) {
+        const videoSender
+            = this.peerconnection.getSenders().find(sender =>
+                sender.track.kind === 'video');
+        const simParams = {
+            encodings: [
+                {
+                    rid: SIM_LAYER_1_RID,
+                    scaleResolutionDownBy: 4
+                },
+                {
+                    rid: SIM_LAYER_2_RID,
+                    scaleResolutionDownBy: 2
```

Comment:
with simulcast each layer is always half of the one above it.  this parameter is in reference to the original resolution, so one is the original resolution, one gets scaled down by a factor of 2 and the last one by a factor of 4